docs/layers.md

```diff
@@ -123,8 +123,6 @@ Combination of OpenStreetMap administrative boundaries (zoom >= 8) and Natural E
 
 #### Boundary properties (optional):
 
-* `labelrank`: from Natural Earth
-* `scalerank`: from Natural Earth
```

Comment:
As we remove this optional `scalerank` property, we also adding a new common property for `min_zoom`, which is sourced from Natural Earth or made up for OSM.